### ISSUE TITLE
Detecting duplicated directory/file entry

### ISSUE THREAD

**Issue by junaruga** [rpm-software-management/rpmlint#613]
I found the following duplicated directory entry `%{ruby_libdir}/io` for the `%files` sections in [the Fedora ruby.spec` file](https://src.fedoraproject.org/rpms/ruby/tree/rawhide) - ruby-3.0.0-146.fc35.src.rpm.

Here is the used rpmlint version.

```
$ rpm -q rpmlint
rpmlint-1.11-15.fc33.noarch
```

```
%files libs
...
%{ruby_libdir}/io
...
%files -n rubygem-io-console
%{ruby_libdir}/io
...
```

I think rpmlint does not detect the issue. I found the issue by the following commands. So, it is great if the rpmlint detects the issue. What do you think?

```
$ rpm2cpio ruby-libs-3.0.0-146.fc35.x86_64.rpm | cpio -idmv

$ rpm2cpio rubygem-io-console-0.5.6-146.fc35.x86_64.rpm | cpio -idmv
...
cpio: ./usr/share/ruby/io not created: newer or same age version exists
...
```
